src/main/java/teammates/common/util/Config.java

```diff
@@ -72,7 +72,8 @@ public static String getAppId() {
      * instead of '-' e.g., "4.53"
      */
     public static String getAppVersion() {
-        return SystemProperty.applicationVersion.get().split("\\.")[0].replace("-", ".");
+        String appVersion = SystemProperty.applicationVersion.get();
+        return appVersion == null ? null : appVersion.split("\\.")[0].replace("-", ".");
```

Comment:
`EmailGeneratorTest` will. `SystemProperty.*.get()` is null if not called in production server.